src/main/java/data/MarkTaskException.java

```diff
@@ -0,0 +1,47 @@
+package data;
```

Comment:
Good job to have a new class for task marking exceptions

Reply:
Seconded! Might want to write some test cases for the new class in the coming days.